modules/EnergyManager/manifest.json

```diff
@@ -8,7 +8,9 @@
     },
     "requires": {
         "energy_trunk": {
-            "interface": "energy"
+            "interface": "energy",
+            "min_connections": 1,
+            "max_connections": 12
```

Comment:
Not sure that we should support multiple energy_trunks here. That would mean totally independent house power connections. The danger is that someone just connects multiple evses to the energymanager here, which would not do what the user expects. I would suggest to leave it at a single energy_trunk and manage only one tree per energymanager. If someone really has two grid connections to his site he can still load two independent energy managers, which would be more explicit

Reply:
That is true. Reduced to single connection.